Mathlib/Analysis/BoxIntegral/Basic.lean

```diff
@@ -194,7 +194,8 @@ theorem hasIntegral_iff : HasIntegral I l f vol y ↔
     ∀ ε > (0 : ℝ), ∃ r : ℝ≥0 → ℝⁿ → Ioi (0 : ℝ), (∀ c, l.RCond (r c)) ∧
       ∀ c π, l.MemBaseSet I c (r c) π → IsPartition π → dist (integralSum f vol π) y ≤ ε :=
   ((l.hasBasis_toFilteriUnion_top I).tendsto_iff nhds_basis_closedBall).trans <| by
-    simp [@forall_swap ℝ≥0 (TaggedPrepartition I)]
+    simp only [exists_and_right, mem_setOf_eq, mem_closedBall, and_imp, forall_exists_index,
+      @forall_swap ℝ≥0 (TaggedPrepartition I)]
```

Comment:
Do you know why this breaks? If it's a timeout, can you mention it in the PR description like the other ones?